.gitattributes

```diff
@@ -1 +1,2 @@
-*.sh text eol=lf
\ No newline at end of file
+*.sh text eol=lf
+*.jsonlines text eol=lf
```

Comment:
This is needed for when we eventually can run all the tests, for the snapshot tests comparison (to avoid it comparing CRLF to LF)